### PR TITLE
Cypress test for auto translate

### PR DESCRIPTION
**Pull request by andrewdotn**
- If logged in, you see an auto-translation for acâhkosa
  - If not logged in, you do not see any auto-translations

### CODE REVIEW

**Comment by andrewdotn** [comment]
While making the tests pass, I cleaned out a bunch of unused scripts and pipenv `[scripts]` entries. Let me know if you’d prefer that to go into a separate PR.

File: cypress/integration/admin.spec.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
+const { join } = require('path');
+
+context('Admin interface', () => {
+  it('should redirect anonymous users to the login page', function() {
+    cy.visit('/admin')
+    cy.location().should('match', /\/admin\/login\//)
```
Comment (dwhieb) [inline cypress/integration/admin.spec.js:6]: Just a little more readable.

```suggestion
    cy.location().should(`contain`, `/admin/login`)
```

File: cypress/integration/admin.spec.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
+const { join } = require('path');
```
Comment (dwhieb) [inline cypress/integration/admin.spec.js:1]: "join" loses its context when it's destructured from `path`—it's not clear what's being joined.

```suggestion
const path = require('path'); // and then just use path.join() elsewhere
```

```suggestion
const { join: joinPathSegments } = require('path');
```

File: cypress/integration/admin.spec.js [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+context('Admin interface', () => {
+  it('should redirect anonymous users to the login page', function() {
+    cy.visit('/admin')
+    cy.location().should('match', /\/admin\/login\//)
+  })
+
+  function login() {
+    cy.visit('/admin')
+    cy.readFile(join(__dirname, '..', '..', 'CreeDictionary', '.cypress-user.json')).then(cypressUser => {
```
Comment (dwhieb) [inline cypress/integration/admin.spec.js:11]:
```suggestion
    cy.readFile(path.join(__dirname, `../../CreeDictionary/.cypress-user.json`)).then(cypressUser => {
```

File: cypress/integration/admin.spec.js [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+      cy.get('#id_username').type(cypressUser.username)
+      cy.get('#id_password').type(cypressUser.password)
+      cy.get('.submit-row > input').click()
+    })
+    cy.location('pathname').should('eq', '/admin/')
```
Comment (dwhieb) [inline cypress/integration/admin.spec.js:16]: Should this assertion be inside the callback for `cy.readFile().then()`? I _think_ the way it is currently, this assertion will resolve as true before Cypress finishes logging in. Or does this timing not matter here?

File: cypress/integration/admin.spec.js [hunk 5]
```diff
@@ -0,0 +1,44 @@
+const { join } = require('path');
+
+context('Admin interface', () => {
+  it('should redirect anonymous users to the login page', function() {
+    cy.visit('/admin')
+    cy.location().should('match', /\/admin\/login\//)
+  })
+
+  function login() {
```
Comment (dwhieb) [inline cypress/integration/admin.spec.js:9]: We might not have to use the user interface to log in for each test, which the Cypress docs treat as an anti-pattern. (Though I vaguely remember you saying that it was difficult to log in programmatically in the app? Just ignore this if so.) We could [add a `cy.login()` command](https://docs.cypress.io/api/cypress-api/custom-commands.html#Syntax) for testing purposes.

Cypress has some useful docs on logging in during tests:

* https://docs.cypress.io/guides/references/best-practices.html#Organizing-Tests-Logging-In-Controlling-State
* https://github.com/cypress-io/cypress-example-recipes#logging-in-recipes

**Review by dwhieb — COMMENTED** [review]
Just some minor comments / suggestions on the Cypress tests.

**Comment by nienna73** [comment]
You might consider moving the URLs into their own settings file to make them easier to manage, otherwise these tests look good!

**Review by dwhieb — APPROVED** [review]

**Pull request merged**